keyboards/kbdfans/kbd6x/keymaps/mekberg/rules.mk

```diff
@@ -0,0 +1,2 @@
+NKRO_ENABLE = yes            # USB Nkey Rollover
+CONSOLE_ENABLE = no        # Console for debug(+400)
```

Comment:
```suggestion
CONSOLE_ENABLE = no        # Console for debug
```

Reply:
Not sure I understand this edit. The same comment is in other contributors' rules files, which seems to be copied from elsewhere in turn.

Reply:
Generally, this number is misleading. It was probably accurate once, but as the codebase (and compiler!) evolves it has and will get more and more outdated. See #6947; there's an ongoing effort to remove them as it's just not practical to keep track of the size costs of all the features. As that PR can't touch user keymaps without becoming a breaking change, I'm going to try to catch as many of them as I can incoming :)

Reply:
alright, thanks for explaining!